### PR TITLE
feat: flutter profiling docs

### PR DESCRIPTION
**Pull request by vaind**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Adds relevant docs for the [Flutter profiling alpha](https://github.com/getsentry/sentry-dart/pull/1611).

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: src/platforms/common/profiling/index.mdx
Comment (shanamatthews) [inline src/platforms/common/profiling/index.mdx:320]:
```suggestion
    // The sampling rate for profiling is relative to tracesSampleRate
    // Setting to 1.0 will profile 100% of sampled transactions:
```

**Review by shanamatthews — APPROVED** [review]
Looks all good to me! Just added one line to your comment to explain a little more clearly, like we do with Python

**Comment by vaind** [comment]
@shanamatthews I've accepted the suggestion but cannot merge the PR.

**Comment by shanamatthews** [comment]
Great. I believe this needs to wait for merge until there's a new flutter release, but I'll merge after that happens. @bruno-garcia is that right?

**Comment by vaind** [comment]
> Great. I believe this needs to wait for merge until there's a new flutter release, but I'll merge after that happens. @bruno-garcia is that right?

In that case: it's already released https://github.com/getsentry/sentry-dart/releases/tag/7.12.0

**Comment by shanamatthews** [comment]
Oh perfect! I'll merge now @vaind

**Pull request merged**
